src/datastore/postgres-store.ts

```diff
@@ -3025,27 +3067,13 @@ export class PgDataStore
       return { total: totalQuery.rows[0].count, rows: resultQuery.rows };
     });
 
-    const parsed = queryResult.rows.map(r => this.parseMempoolTxQueryResult(r));
-    return { results: parsed, total: queryResult.total };
-  }
-
-  async getMempoolTxIdList(): Promise<{ results: DbMempoolTxId[] }> {
-    return this.query(async client => {
-      const resultQuery = await client.query<MempoolTxIdQueryResult>(
-        `
-        SELECT ${MEMPOOL_TX_ID_COLUMNS}
-        FROM mempool_txs
-        ORDER BY receipt_time DESC
-        `
-      );
-      const parsed = resultQuery.rows.map(r => {
-        const tx: DbMempoolTxId = {
-          tx_id: bufferToHexPrefixString(r.tx_id),
-        };
-        return tx;
-      });
-      return { results: parsed };
```

Comment:
replying to: @kyranjamie https://github.com/blockstack/stacks-blockchain-api/pull/629#discussion_r650768865
That rosetta mempool endpoint had implemented a redundant SQL here that does the equivalent of a no-limit query. I removed the redundant function, and replaced with the `getMempoolTxList({ limit: Number.MAX_SAFE_INTEGER })` call so the (bad) implications of this API are more clear.